app/assets/stylesheets/events.scss

```diff
@@ -1,3 +1,65 @@
 // Place all the styles related to the Events controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+// FONT
+@font-face {
+    font-family: Gotham-Book;
+    src: url(../fonts/Gotham-Book.otf);
+}
+
+@font-face {
+    font-family: Gotham-Light;
+    src: url(../fonts/Gotham-Light.ttf);
+}
+
+@font-face {
+    font-family: Gotham-Medium;
+    src: url(../fonts/Gotham-Medium.otf);
+}
+
+// VARIABLES
+
+$light: #ffffff;
+$dark-gray: #292929;
+$mid-gray: #4A4A4A;
+$light-gray: #9B9B9B;
+$highlight: #F9ED00;
+$border-opacity: rgba(155, 155, 155, .3);
```

Comment:
I think this should be renamed to `$border-color` as it is a color. I think everything else looks great.